threestudio/data/image.py

```diff
@@ -29,8 +31,9 @@
 
 @dataclass
 class SingleImageDataModuleConfig:
-    height: int = 96
-    width: int = 96
+    height: Any = 96
+    width: Any = 96
```

Comment:
Instead of `height: Any`, it would be best to use `height:Union[int,List[int]]` or whatever types you expect. You might need to add a `from typing import Union` on top.

Reply:
Ideally yes, but OmegaConf does not support Union of containers. Will mention this in a comment